frontend/src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -286,7 +286,7 @@ class ServiceDetailsPageComponent extends React.Component<ServiceDetailsProps, S
 }
 
 const mapStateToProps = (state: KialiAppState) => ({
-  jaegerInfo: state.jaegerState.info,
+  jaegerInfo: state.tracingState.info,
```

Comment:
If the state is generic (tracingState), the variable should be renamed to tracingInfo.